online-shop/src/app/components/containers/product-list/product-list.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import { products } from 'src/app/mocks/products.mocks';
+import { Product } from 'src/app/modules/shared/types/products.types';
+@Component({
+  selector: 'app-product-list',
+  templateUrl: './product-list.component.html',
+  styleUrls: ['./product-list.component.scss'],
+})
+export class ProductListComponent {
+  productss: Product[] = products;
```

Comment:
Here is the "root" problem of the typo :D

Reply:
Done